cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -307,51 +307,70 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
       p("include::dbms/admin-role-dbms.asciidoc[]")
 
       section("Using a custom role to manage DBMS privileges", "administration-security-administration-dbms-custom", "enterprise-edition") {
-        p("include::dbms/admin-role-dbms-custom.asciidoc[]")
-        p("First we copy the 'admin' role:")
-        query("CREATE ROLE usermanager AS COPY OF admin", ResultAssertions(r => {
-          assertStats(r, systemUpdates = 2)
+        p(
+          """If it is desired to have an administrator with a subset of privileges that includes all DBMS privileges, but not all database privileges, this can be achieved in multiple ways.
+            |One way is to copy the `admin` role and revoking or denying the unwanted privileges.
+            |A second version is to build the custom administrator from scratch by granting the wanted privileges instead.
+            |""".stripMargin)
+        p("As an example, lets create an administrator using the second method that can only manage users and roles.")
+        p("First we create the new role:")
+        query("CREATE ROLE usermanager", ResultAssertions(r => {
+          assertStats(r, systemUpdates = 1)
         })) {
           statsOnlyResultTable()
         }
-        p("Then we DENY ACCESS to normal databases:")
-        query("DENY ACCESS ON DATABASE * TO usermanager", ResultAssertions(r => {
+        p("Then we GRANT USER MANAGEMENT:")
```

Comment:
I think this could be written more as what the command means instead of just the command.
Like:
```suggestion
        p("Then we grant the privilege to manage users:")
```

Reply:
I just followed what was before, but agree yours is better